Source/PeleLM_derive.H

```diff
@@ -58,6 +58,22 @@ void pelelm_dergrdpz (const amrex::Box& bx, amrex::FArrayBox& derfab, int dcomp,
                         amrex::Real time, const int* bcrec, int level);
 #endif
 
+void pelelm_dertransportcoeff (const amrex::Box& bx, amrex::FArrayBox& derfab, int dcomp, int ncomp,
+                        const amrex::FArrayBox& datafab, const amrex::Geometry& geomdata,
+                        amrex::Real time, const int* bcrec, int level);
+
+void pelelm_dermixanddiss (const amrex::Box& bx, amrex::FArrayBox& derfab, int dcomp, int ncomp,
+                        const amrex::FArrayBox& datafab, const amrex::Geometry& geomdata,
+                        amrex::Real time, const int* bcrec, int level);
+
+void pelelm_dermixfrac (const amrex::Box& bx, amrex::FArrayBox& derfab, int dcomp, int ncomp,
+                        const amrex::FArrayBox& datafab, const amrex::Geometry& geomdata,
+                        amrex::Real time, const int* bcrec, int level);
+
+void pelelm_dhrr (const amrex::Box& bx, amrex::FArrayBox& derfab, int dcomp, int ncomp,
```

Comment:
where is pelelm_dhrr ? I did not find it?

Reply:
Working on it. There is just the header right now. I sorta pulled a PR because I saw all these other PR being merged and I already had to merge dev in my branch twice...

Reply:
I understand, but can we focus on committing clean and finished things inside development, and not WIP stuff ?

Reply:
More of a philosophical question, you ask me ... Anyways, my last commit should make you happy for now.